build.sbt

```diff
@@ -595,6 +600,31 @@ def isInTestShard(name: String, logger: Logger): Boolean = {
   }
 }
 
+lazy val metalsDependencies = project
+  .in(file("target/.dependencies"))
+  .settings(
+    publish / skip := true,
+    excludeDependencies ++= Seq(
+      // we are not interested in downloading scalafix here
+      ExclusionRule("ch.epfl.scala", "scalafix-core_2.13")
+    ),
+    libraryDependencies ++= List(
+      // The dependencies listed below are only listed so Scala Steward
+      // will pick them up and update them. They aren't actually used.
```

Comment:
I'm wondering if we can just use `latest.release` for scalafix rules in Embedded instead of exact build-in version

Reply:
Good idea, let me check that. Totally forgot about it, we wouldn't need to rely on scala steward

Reply:
That works, thanks!